server/config/policies.js

```diff
@@ -36,7 +36,7 @@ module.exports.policies = {
   "forms/post-contact-us-form": true,
 
   // Cache
-  "cache/*": ["isLoggedIn", "isAdmin"],
+  "cache/*": ["isLoggedIn", "isTech"],
```

Comment:
https://hongkong.hmcc.net/clear-cache is the current link to clear cache in the website. so should change this to /clear-cache